src/appkit.cs

```diff
@@ -10535,6 +10535,18 @@ interface NSLayoutXAxisAnchor
 		[Mac (10,12)]
 		[Export ("anchorWithOffsetToAnchor:")]
 		NSLayoutDimension GetAnchorWithOffset (NSLayoutXAxisAnchor otherAnchor);
+
+		[Mac (10,16)]
+		[Export ("constraintEqualToSystemSpacingAfterAnchor:multiplier:")]
+		NSLayoutConstraint ConstraintEqualToSystemSpacingAfterAnchor (NSLayoutXAxisAnchor anchor, nfloat multiplier);
+
+		[Mac (10,16)]
+		[Export ("constraintGreaterThanOrEqualToSystemSpacingAfterAnchor:multiplier:")]
+		NSLayoutConstraint ConstraintGreaterThanOrEqualToSystemSpacingAfterAnchor (NSLayoutXAxisAnchor anchor, nfloat multiplier);
+
+		[Mac (10,16)]
+		[Export ("constraintLessThanOrEqualToSystemSpacingAfterAnchor:multiplier:")]
+		NSLayoutConstraint ConstraintLessThanOrEqualToSystemSpacingAfterAnchor (NSLayoutXAxisAnchor anchor, nfloat multiplier);
```

Comment:
I understand the naming.. but is so verbose! do we need, I'd do:
```suggestion
		NSLayoutConstraint EqualToSystemSpaceAfterAnchor (NSLayoutXAxisAnchor anchor, nfloat multiplier);

		[Mac (10,16)]
		[Export ("constraintGreaterThanOrEqualToSystemSpacingAfterAnchor:multiplier:")]
		NSLayoutConstraint GreaterThanOrEqualToSystemSpaceAfterAncho (NSLayoutXAxisAnchor anchor, nfloat multiplier);

		[Mac (10,16)]
		[Export ("constraintLessThanOrEqualToSystemSpacingAfterAnchor:multiplier:")]
		NSLayoutConstraint LessThanOrEqualToSystemSpaceAfterAncho (NSLayoutXAxisAnchor anchor, nfloat multiplier);
```

Reply:
Unfortunately we can't do that, maybe for XAMCORE_4_0. This code exist on iOS too using the same names so we can increase code sharing between the platforms.